vagrant/provision/provision-0.sh

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env bash
```

Comment:
This is the vagrant provisioning script that installs all of the pre-requisites and configures the server from bare metal to a working state. After development this can be deconstructed into a non-vagrant specific installation script.